open_dread_rando/patcher_editor.py

```diff
@@ -55,7 +55,8 @@ def ensure_present_in_scenario(self, scenario: str, asset):
     def resolve_actor_reference(self, ref: dict) -> Container:
         scenario = self.get_scenario(ref["scenario"])
         layer = ref.get("layer", "default")
-        return scenario.actors_for_layer(layer)[ref["actor"]]
+        actor = scenario.actors_for_layer(layer)
```

Comment:
actors

Reply:
I'll revert this file to the previous commit, it was a remnant of a debug print from earlier testing.